utility/BUILD.bazel

```diff
@@ -0,0 +1,53 @@
+load("@io_bazel_rules_kotlin//kotlin:kotlin.bzl", "kt_android_library")
+load("@rules_jvm_external//:defs.bzl", "artifact")
+load("@dagger//:workspace_defs.bzl", "dagger_rules")
+
+package(default_visibility = ["//visibility:public"])
+
+kt_android_library(
+  name = "utility_lib",
+  custom_package = "org.oppia.util",
+  srcs = [
+      "src/main/java/org/oppia/util/accessibility/*.kt",
```

Comment:
As mentioned, can we glob this to make sure that all Kotlin files are included?

Reply:
Done :)